feat: instructions for passkey mobile configuration (#55)
rownd/docs: configuration/mobile/ios.mdx

description: "Preparing Rownd to work with your iOS app"
---

> Configure Rownd for iOS mobile devices including ([React Native](/sdk-reference/mobile/react-native), [Flutter](/sdk-reference/mobile/flutter)). This configuration will allow multiple Rownd services to work in the iOS operating system like [Universal Links](/configuration/mobile/overview#what-are-universal-links) as well [Passkeys](/configuration/authentication-methods/passkeys).

Reviewer: ```suggestion
Configure Rownd for iOS mobile devices including ([React Native](/sdk-reference/mobile/react-native) and [Flutter](/sdk-reference/mobile/flutter)). This configuration will allow multiple Rownd services to work in the iOS operating system like [Universal Links](/configuration/mobile/overview#what-are-universal-links) and [Passkeys](/configuration/authentication-methods/passkeys).
```
